frontend/src/app/create/page.js

```diff
@@ -1,89 +1,70 @@
-"use client";
```

Comment:
There shouldn't be any changes in this file

Reply:
I would probably just copy and paste what's currently in `main` for this file

Reply:
Changes were prettier formatting

Reply:
I think right now it's undoing prettier formatting. the changes might've been from before you pulled `main` in because it doesn't contain the fix that was merged in (currently missing the `credentials: "include"`)

```
const options = {
      method: "POST",
      headers: {
        "Content-Type": "application/json",
      },
      body: data,
      credentials: "include",
    };
```